src/utils/generateContent.ts

```diff
@@ -0,0 +1,41 @@
+import hbs from 'handlebars';
+
+import type { JsonValue } from '@/types/Template';
+import { TemplateType } from '@/types/Template';
+
+type TemplateParams = {
+  templateType: TemplateType;
+  templateContent: string;
+  templateStyle?: string;
+  templateData?: JsonValue;
+};
+
+const generateTemplateContent = ({
+  templateType,
+  templateContent,
+  templateStyle = '',
+  templateData = {},
+}: TemplateParams): string => {
+  if (templateType === TemplateType.HANDLBARS_TEMPLATE) {
+    const compiledTemplate = hbs.compile(templateContent);
+    return compiledTemplate(templateData);
+  }
+
+  // For regular HTML templates
+  return `
```

Comment:
before returning this, do a lint check for style as well as templateContent (html)